File: node/1.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});
readline.question('Введите температуру на улице: ', (i) => {
    const rad = parseInt(i);
    if (rad < 0) {
        console.log('Холодно! Одевайся потеплее!');
    } else {
        if (rad > 20) {
            console.log('На улице тепло, можно надеть футболку.');
        } else {
            console.log('Прохладно, лучше взять куртку.');
        }
    }
    readline.close();
})

File: rust/1.rs
use std::io;

fn main() {
    println!("Введите температуру на улице: ");

    let mut input = String::new();
    io::stdin()
        .read_line(&mut input)
        .expect("Ошибка чтения ввода");
    let rad: i32 = input
        .trim()
        .parse()
        .expect("Введено не число");
    if rad < 0 {
        println!("Холодно! Одевайся потеплее!");
    } else if rad > 20 {
        println!("На улице тепло, можно надеть футболку.");
    } else {
        println!("Прохладно, лучше взять куртку.");
    }
}